dm/worker/status.go

```diff
@@ -129,25 +131,27 @@ func (w *Worker) StatusJSON(stName string) string {
 func (w *Worker) Error(stName string) []*pb.SubTaskError {
 	w.Lock()
```

Comment:
same with `w.Status`

Reply:
removed in https://github.com/pingcap/dm/pull/259/commits/01f693ba86374689365439155952183820910bb5.